### ISSUE TITLE
New report for hubs to know how much customers owe to each producer

### ISSUE THREAD

**Issue by nicolasVr** [openfoodfoundation/wishlist#519]
This is a proposal for a new report that is currently not available.
Some hubs are organized in such a way that each customer pay directly each producer. For them it would be be very handy to generate a report for a dedicated cycle that display for each customer how much he should pay to the producers : 

Customer Name | Producer Name | Total Price to pay

**Comment by BethanOFN** [comment]
Hi @nicolasVr, this could easily be done as a custom report integration in Google Sheets/n8n. It would be about 4hrs work @£50 per hour, plus VAT. Let me know if that would be of interest 👍

**Comment by nicolasVr** [comment]
Thanks @BethanOFN , I am not very fond of Google tools, butI think anyway that https://github.com/openfoodfoundation/openfoodnetwork/issues/12776 should answer this request.

**Comment by tschumilas** [comment]
I think there could be 2 different things:    1.  the report being developed #12766 -  will tell the hub manager how much each supplier/producer needs to be paid.  The hub manager will then send this money in whatever fashion they usually do.  But customers (buyers) are buying from the hub,  not from individual producers.       2.  the issue I thought you were asking about @nicolasVr  was a situation where the customer/buyer  pays each supplier/producer directly.   So there is a hub store with multiple suppliers.  A customer checks out buying products from several producers.   Payment is NOT made at checkout.  After checkout,  an integration is used (aka @BethanOFN )  to send a composite invoice to the customer that details how much the customer should pay EACH supplier/producer,  and perhaps HOW they should pay.     So in effect this is a 'split payment' hack.  Would be very useful here - situations where the 'hub'  is really just an informal network of suppliers and not an official organization.  So there is no bank account for the hub to receive payments.

**Comment by nicolasVr** [comment]
You are right @tschumilas , it is different. Bug #12835 is driving me crazy !
Anyway, can we avoid using Google Sheets ?
